packages/pixel-demo/test/swingsetTests/gallery/vat-alice.js

```diff
@@ -102,7 +102,12 @@ function makeAliceMaker(E, log, contractHost) {
           showPaymentBalance('childPayment2', childPayment2);
 
           const bobsRawPixel = result.extent[0];
-          insist(
+          // TODO this was previously unchecked
```

Comment:
Before my change, it was in fact an assert. Kate changed it to a console log to see what was up. Up to you two whether there is an assertion there to be checked.

Reply:
@katelynsills what is this for? What should we do with this?

Reply:
I think we should drop the assertion rather than spend time on fixing the pixel demo. That makes me think we should move it to Agoric-labs so monorepo-wide changes don't require us to dig into the pixel demo...